.github/workflows/ubuntu.yml

```diff
@@ -3,28 +3,27 @@ name: ubuntu
 
 on: [push, pull_request]
```

Comment:
I think all jobs should immediately abort if the target repo is not "aibasel/downward". Then users who want to run (a subset of) the tests can do so manually.

Reply:
Do you know how to do this? I didn't find any events that care about the repository. Also, I don't think the solution to abort is nice because it will still run actions on forks and create lots of emails saying that an action failed.

Reply:
I found a page that says we can add an if condition to the jobs. I tested it on my work in branch GHATest (use != for not equal). This creates an entry for the action, but the action is denoted as skipped and no mail is send

```
jobs:
  compile: # name of the job. Jobs run in parallel unless specified otherwise.
    name: compile
    if: github.repository == 'PatrickFerber/downward'
```
https://github.community/t/stop-github-actions-running-on-a-fork/17965

Reply:
Thanks for looking this up. I found something along those lines before. I just don't think it makes sense to encode into the workflows themselves that they should be run/not run on specific repositories because a user then would need to change the workflows to enable them which doesn't make sense in my opinion.

Reply:
I still think having the tests be opt-in is nicer and less surprising than opt-out. If a user wants them they need adapting anyway because otherwise the LP tests will fail.

Reply:
That's not right: the LP compilation will be skipped. I think the whole thing warrants another discussion.

Reply:
In this PR, yes, it will be skipped. But is this desirable? In issue1005 we're planning to make this an error. Otherwise we might miss a failed LP compilation.

Reply:
We already discussed this when Florian got the LP errors because he hadn't set up the secrets. I think this is a good solution, yes. Note that when we merge the branch, the LP solvers will be tested even if the person creating the PR does not test them because she doesn't have the secrets.